instrumentation/vertx-core-4.1.0/src/main/java/com/nr/vertx/instrumentation/VertxCoreUtil.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *  * Copyright 2023 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.nr.vertx.instrumentation;
+
+import com.newrelic.agent.bridge.AgentBridge;
+import com.newrelic.api.agent.GenericParameters;
+import com.newrelic.api.agent.HttpParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Segment;
+import com.newrelic.api.agent.Token;
+import com.newrelic.api.agent.Transaction;
+import com.newrelic.api.agent.weaver.Weaver;
+import io.vertx.core.Handler;
+import io.vertx.core.http.impl.HttpClientResponseImpl;
+import io.vertx.core.impl.future.Listener;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+
+public class VertxCoreUtil {
+
+    private VertxCoreUtil() {
+    }
+
+    private static final Map<Object, Token> tokenMap = AgentBridge.collectionFactory.createConcurrentWeakKeyedMap();
+
+    public static final String VERTX_CLIENT = "Vertx-Client";
+    public static final String END = "handleResponse";
+
+    private static final URI UNKNOWN_HOST_URI = URI.create("http://UnknownHost/");
+
+    public static void storeToken(Listener handler) {
```

Comment:
Nit picking, but can you change the variable name to listener, so it matches the class and the name in the 4.3.2 module.